R/pkg/R/backend.R

```diff
@@ -25,9 +25,23 @@ isInstanceOf <- function(jobj, className) {
   callJMethod(cls, "isInstance", jobj)
 }
 
-# Call a Java method named methodName on the object
-# specified by objId. objId should be a "jobj" returned
-# from the SparkRBackend.
+#' Call Java Methods
+#'
+#' Call a Java method in the JVM running the Spark driver.
+#'
+#' @param objId object to invoke the method on. Should be a "jobj" created by newJObject.
+#' @param methodName method name to call.
+#' @param ... parameters to pass to the Java method.
+#' @export
+#' @seealso callJStatic, newJObject
+#' @examples
```

Comment:
roxygen2 infers the rdname pretty well, so I agree that is largely optional.
From a best practice/guideline point-of-view though I think it'd be good to have `@rdname`  because it would make it clear which Rd and doc page it goes to. I think it would get people to think about it.

For example, instead of `n`->`n` by default we want `n`->`count` since `n` is an alias of `count`.

As for @aliases, agree we don't need it. On a related note, I think I'm seeing some issues with it in other places we have that I want to look into shortly. Let me check back with you later.